kubernetes-manifests/adservice.yaml

```diff
@@ -16,7 +16,6 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: adservice
-  namespace: demo-env
```

Comment:
maybe leave it in namespace demo-env? otherwise it would go to default
perhaps you can ensure `demo-env` exists as part of setup

Reply:
yes we can, but need to add a small extra step to create the namespace